projects/js-packages/shared-extension-utils/src/hooks/use-analytics.js

```diff
@@ -1,11 +1,36 @@
 import jetpackAnalytics from '@automattic/jetpack-analytics';
 import { useConnection } from '@automattic/jetpack-connection';
 import { useEffect } from '@wordpress/element';
+import { useState, useCallback } from 'react';
```

Comment:
Maybe we could import the abstractions from `@wordpress/element`, like we do above for `useEffect`?